docs/api-docs/payments/payments-api-overview.md

```diff
@@ -197,7 +197,7 @@ lineNumbers: true
 **Get payment methods = process payment**
 * type = type
 * token = token
-* last_four = verification_value
+* CVC = verification_value
```

Comment:
I believe this is showing the mapping of fields returned in `GetPaymentMethods` call to be used in processing payments.
[As per the docs for GetPaymentMethods call](https://developer.bigcommerce.com/api-reference/store-management/payment-processing/accepted-methods/paymentsmethodsget), we won't have CVC information returned from this call.

```suggestion
```

Reply:
Fair point, that might have been confusing. I've removed this and added @IamDavidovich's suggested description of this field instead